internal/testrunner/runners/pipeline/runner.go

```diff
@@ -192,9 +194,75 @@ func (r *runner) run(ctx context.Context) ([]testrunner.TestResult, error) {
 		results = append(results, result)
 	}
 
+	if r.options.SkipCheckPipelineWarnings {
+		return results, nil
+	}
+
+	esLogs, err := r.checkElasticsearchLogs(ctx, startTime)
+	if err != nil {
+		return nil, err
+	}
+	results = append(results, esLogs...)
+
 	return results, nil
 }
 
+func (r *runner) checkElasticsearchLogs(ctx context.Context, startTesting time.Time) ([]testrunner.TestResult, error) {
+
+	startTime := time.Now()
+
+	testingTime := startTesting.Truncate(time.Second).Add(-1 * time.Second)
```

Comment:
Could you add a comment about the reason for this adjustment?

Reply:
thanks for catching that, this (-1 sec) is not necessary, I was hunting down the cisco_ftd not being detect with warning in the integrations test but it was the alternative format log type the reason. Removed 🙂